src/main/java/com/project/workingtime/CheckController.java

```diff
@@ -0,0 +1,24 @@
+package com.project.workingtime;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@RestController
+public class CheckController {
+
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
+    LocalDateTime now = LocalDateTime.now();
+
+    @RequestMapping("/check-in")
+    public String checkIn() {
+        return "Check in " + dtf.format(now);
```

Comment:
There is a bug in your app. 
I started the server today morning. At this moment, I'm sending a check-in request that is returning the same time that I had in the morning.